betel/data_set_builder.py

```diff
@@ -0,0 +1,76 @@
+import pathlib
+import abc
+from typing import Dict
+import pandas as pd
+from sklearn import model_selection
+from betel import utils
+from betel import info_files_helpers
+
+
+class DataSetBuilder(metaclass=abc.ABCMeta):
+    """A class for splitting data into train-validation-test sets."""
+
+    _RANDOM_SEED = 2579  # seed for random splitting
+
+    def __init__(self, input_dir: pathlib.Path, storage_dir: pathlib.Path,
+                 split_ratio: (float, float, float) = (0.7, 0.15, 0.15)):
+        """"Constructor.
+
+        :param input_dir: data to be split (output of the scraper)
```

Comment:
Please explicitly state what's expected to be in the input_dir. It should be clear to the reader what the output of the scraper is